src/helpers/Env.helper.js

```diff
@@ -0,0 +1,7 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const env = (param, value) => process.env[param] || value;
+
+export default env;
```

Comment:
What is the use of this helper? Could you explain ?

Reply:
**What it does is**
- Instead of using like process.env.DATABASE_URL for calling our environment variables I created
**How it is used**  
- You import like this: ```import env from '../../helpers/Env.helper';```
- Instead of using process.env.DATABASE_URL we can use ```env('DATABASE_URL');'```